### ISSUE TITLE
Create a warning inside plugin configuration when Article's manager parameter "Show article options" is disabled

### ISSUE THREAD

**Issue by ggppdk** [vistamedia/socialmeta#5]
Check and show messages inside plugin configuration, e.g. check **Article Manager** parameter **show_article_options** which will prevent SocialMeta TAB from showing inside article form

---
- Either create a new form element (parameter) (for better placement):

```
class JFormFieldSocialMetaChecks extends JFormFieldList
```
- or via enqueueing a message on as system event of the plugin

```
JFactory::getApplication()->enqueueMessage('..', 'warning');
```

---

So to avoid user wondering why **social TAB** is not showing when:

```
JComponentHelper::getParams( 'com_content' )->get('show_article_options') == 0
```

We need to add a warning message like:

> 'Article manager' parameter: "Show article options", needs to be enabled otherwise,
> plugin's 'Social' parameter TAB , will not be shown in the article form

**Comment by vistamedia** [comment]
Agreed, I didn't even know this option. I will add a notice for the superusers.
